driver/bpf/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM debian:unstable
```

Comment:
I believe the path chosen for all these files (`driver/bpf/`) might not be the best one, I see no point in setting the build context of Docker to this directory, it makes you think the Docker build will somehow need the other source files at this location, which it doesn't. The best path for this would probably be one of `ROOT/docker/ebpf-probe-builder`, `ROOT/test/ebpf-probe-builder`, `ROOT/scripts/ebpf-probe-builder`. Further, these scripts seem to be hardcoded to work with the agent (e.g. `ENV AGENT_VERSION 0.1.1dev` here and `/opt/draios/src/draios-agent-0.1.1dev` later), so perhaps they should be moved in the agent repository or properly refactored to make sure they can be used against sysdig as well.